### PR TITLE
Added a Ruby example according to the issue 'Ruby on rails example #344'

### PR DESCRIPTION
**Pull request by Gauravsaha-97**

### CODE REVIEW

File: examples/c/src/sum.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+  printf("Enter two numbers to add: \n");
+  scanf("%d %d", &x, &y);
+
+  z = x + y;
+
+  printf("Sum of the numbers : %d\n", z);
+
+  return 0;
+}
```
Comment (vladaionescu) [inline examples/c/src/sum.c:14]: This seems unrelated?

Comment (Gauravsaha-97) [reply examples/c/src/sum.c:14]: @vladaionescu I have added 'prime_check.rb' file to the 'ruby' directory. This is the C file that you are viewing.

**Comment by RajivSah** [comment]
There is already a ruby example in the repo.

**Comment by agbell** [comment]
Hi @Gauravsaha-97, You have two commits here but only should be part of this PR.  Please remove 6e5621572d70747f7b3ca5d07b653add5286b35e. You could do this with git rebase -i, or just start new branch off master and reintroduce your changes.  Also we are looking for a more fully-featured ruby on rails example.

File: examples/c/src/sum.c (hunk 1 above)
Comment (vladaionescu) [reply examples/c/src/sum.c:14]: Right.. the question is why is this in this PR
